yt_dlp/extractor/tiktok.py

```diff
@@ -511,23 +535,31 @@ def _real_extract(self, url):
         try:
             return self._extract_aweme_app(video_id)
         except ExtractorError as e:
-            self.report_warning(f'{e}; Retrying with webpage')
+            self.report_warning(f'{e.orig_msg}; Retrying with webpage')
 
-        # If we only call once, we get a 403 when downlaoding the video.
-        self._download_webpage(url, video_id)
-        webpage = self._download_webpage(url, video_id, note='Downloading video webpage')
+        # If UA starts with `Mozilla/5.0`, but shorter than 26 character, 403 response is received
+        webpage = self._download_webpage(
+            url, video_id, headers={'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:98.0) Gecko/20100101 Firefox/98.0'})
```

Comment:
```suggestion
        # If we only call once, we get a 403 when downlaoding the video.
        webpage = self._download_webpage(url, video_id, tries=2)
```
I'm stashing this, when `random_user_agent` gets fixed.
Tho, it seams to be working with a single request atm.